test/setup/deployers/types.ts

```diff
@@ -1,9 +1,6 @@
 export interface IDeployer {
-  deploy: (
-    integrationId: string,
-    imageOpts: {
-      imageNameAndTag: string;
-      imagePullPolicy: string;
-    },
-  ) => Promise<void>;
+  deploy: (imageOpts: {
```

Comment:
consider adding a type for that
something like
```imageOptions: {
  nameAndTag: string;
  pullPolicy: string;
}```
or even split the name and tag if appropriate